### DISCUSSION TITLE
Public OQRS without login option

### DISCUSSION THREAD

**Discussion by pa8s** [magicbug/Cloudlog#1905] [Ideas / Feature Requests]
Is it possible, perhaps through a configurable option (Admin -> Global Options?), to get the OQRS page without a login option? This way, via a reverse-proxy, the OQRS page could be accessed without making the entire log/cloudlog site available. Or is something like this already possible via a public-slug?

**Comment by magicbug** [comment]
I'd suggest trying the dev build. https://github.com/magicbug/Cloudlog/tree/dev

QORS is now shown via public slugs.

**Reply by pa8s** [reply]
Already on dev. If I'm logged out I can access the OQRS page (https://cloudlog/index.php/oqrs) but I was wondering if the 'Login' button could be removed with a config option so others could access the OQRS page without the ability to log in.

**Reply by phl0** [reply]
So only hide it on the OQRS page? Would be possible I guess.

**Reply by phl0** [reply]
I made a patch here: https://github.com/phl0/Cloudlog/tree/hideLogin.
But the user would be redirected if he clicks on the "Cloudlog" link in the top left corner anyway.
What do you think @magicbug @AndreasK79? Should I PR this?

**Reply by AndreasK79** [reply]
Having a login is not the same as people having access. They still need a valid login. Removing login button is ok, but leave the Cloudlog link as is.

**Reply by phl0** [reply]
> Removing login button is ok, but leave the Cloudlog link as is.

Sure. In my branch only the login button is removed.

**Reply by magicbug** [reply]
Dont mind having login removed on the public URL

**Reply by phl0** [reply]
PR: https://github.com/magicbug/Cloudlog/pull/1908

**Reply by magicbug** [reply]
Merged

**Reply by pa8s** [reply]
Thank you. :)
